applications/asset_tracker_v2/overlay-azure.conf

```diff
@@ -10,14 +10,11 @@ CONFIG_AZURE_IOT_HUB_AUTO_DEVICE_TWIN_REQUEST=y
 CONFIG_AZURE_IOT_HUB_DEVICE_ID_APP=y
 CONFIG_AZURE_IOT_HUB_MQTT_RX_TX_BUFFER_LEN=2048
 CONFIG_AZURE_IOT_HUB_STACK_SIZE=4096
-CONFIG_AZURE_FOTA=y
-CONFIG_AZURE_FOTA_TLS=y
```

Comment:
**question:** why is this removed?

Reply:
Same for sec tag below.

Reply:
These are no longer necessary. Azure FOTA is auto-enabled as long as the fw is build for a 91 compatible board and the fota download library is enabled.